setup.py

```diff
@@ -2,11 +2,12 @@
 
 
 setup(
-    name="certbot",
+    name='marathon-acme',
     version='0.0.1',
     license='MIT',
-    url="https://github.com/praekeltfoundation/certbot",
-    description="A robot for managing Let's Encrypt! certs in Seed Stack",
+    url='https://github.com/praekeltfoundation/marathon-acme',
+    description='Automated management of Let\'s Encrypt certificates for apps '
+                'running on Mesosphere Marathon',
```

Comment:
Rather use `"` here to avoid the escaping and add parens around the implicit string concatenation to make it clearer that it's deliberate.